### PR TITLE
Int 666 move person race form view to react wood duck

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**

### CODE REVIEW

File: src/test/Race/RaceForm_test.js [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+    expect(
+      component.find('RaceField[race="Unknown"]').props().disabled
+    ).toEqual(false);
+    expect(
+      component.find('RaceField[race="Abandoned"]').props().disabled
+    ).toEqual(true);
+    expect(
+      component.find('RaceField[race="Declined to answer"]').props().disabled
+    ).toEqual(true);
+  });
```
Comment (edgibbs) [inline src/test/Race/RaceForm_test.js]: line feeds between 'it' blocks here to be consistent.

File: src/test/Race/RaceForm_test.js [hunk 2]
```diff
@@ -0,0 +1,383 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import RaceForm from '../../Race/RaceForm';
+import './../EnzymeSetup';
+
+describe('Race', () => {
```
Comment (edgibbs) [inline src/test/Race/RaceForm_test.js]: describe RaceForm

File: src/common/CheckboxField.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import ErrorMessages from '../common/ErrorMessages';
```
Comment (edgibbs) [inline src/common/CheckboxField.js]: So big question here is this a new component that should have its own storybook representation?

File: src/Race/RaceFormContainer.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+const raceDetailOptions = Object.keys(RACE_DETAILS).reduce(
+  (raceDetails, race) => ({
+    ...raceDetails,
+    [race]: RACE_DETAILS[race].map(value => ({ label: value, value })),
+  }),
+  {}
+);
+
+const personId = '1';
```
Comment (edgibbs) [inline src/Race/RaceFormContainer.js]: Is this a reasonable default?

File: src/Race/RaceFormContainer.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```
Comment (edgibbs) [inline src/Race/RaceFormContainer.js]: So is this just for purposes of showing the storybook story with real state?

Comment (NagaPradeepGajula-osi) [reply src/Race/RaceFormContainer.js]: yes the purpose of this container component is just for showing the storybook story.

File: src/common/CheckboxField.js (hunk 3 above)
Comment (NagaPradeepGajula-osi) [reply src/common/CheckboxField.js]: So in Intake this component works based on the calculated data and the story I am working on doesn't say anything about showing the supporting components to be displayed in storybook.

File: src/Race/RaceFormContainer.js (hunk 4 above)
Comment (NagaPradeepGajula-osi) [reply src/Race/RaceFormContainer.js]: I am just giving it a default string for storybook purpose. I believe these values are actually passed as props in intake from the api calls.

File: src/test/Race/RaceForm_test.js (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply src/test/Race/RaceForm_test.js]: done

**Comment by edgibbs** [comment]
@kfinlaysonosi gave her approval, so I'm merging this.

**Pull request merged**